Fix several issues in webext storage docs (#43924)
mdn/content: files/en-us/mozilla/add-ons/webextensions/api/storage/storagearea/index.md

sidebar: addonsidebar
---

> StorageArea is an object representing a storage area. This documentation describes the common interface of the following concrete implementations:

Reviewer: ```suggestion
StorageArea is an object representing a storage area. This documentation describes the common interface of these concrete implementations:
```

- {{WebExtAPIRef("storage.local")}}
- {{WebExtAPIRef("storage.managed")}}
- {{WebExtAPIRef("storage.session")}}
- {{WebExtAPIRef("storage.sync")}}

> `storage.StorageArea` is not an actual API object, it is merely a mechanism to describe the above APIs.

Reviewer: ```suggestion
`storage.StorageArea` is not an API object; it is a mechanism for describing the concrete APIs.
```
